lib/PlacesProvider.js

```diff
@@ -194,17 +194,27 @@ Links.prototype = {
       limit = HISTORY_RESULTS_LIMIT;
     }
 
-    let sqlQuery =  `SELECT moz_places.url as url, moz_favicons.data as favicon, mime_type as mimeType,
-                            title, frecency, last_visit_date as lastVisitDate,  "history"  as type
+    let sqlQuery =  `SELECT moz_places.url as url,
+                            moz_favicons.data as favicon,
+                            moz_favicons.mime_type as mimeType,
+                            moz_places.title,
+                            moz_places.frecency,
+                            moz_places.last_visit_date as lastVisitDate,
+                            "history"  as type,
+                            moz_bookmarks.guid as bookmarkGuid,
+                            moz_bookmarks.dateAdded / 1000 as bookmarkDateCreated
```

Comment:
Agreed, and we also need to change the tests to insert microseconds, which i did in the this PR https://github.com/mozilla/activity-streams/pull/203.